src/root/components/AuthenticatedLayout/Breadcrumbs/index.js

```diff
@@ -1,10 +0,0 @@
-import withBreadcrumbs from 'react-router-breadcrumbs-hoc';
```

Comment:
Is this dep removed from package.json?

Reply:
Whoops, I added a commit to remove this package, but I forgot to push it up.  I'll do that. 😅

Reply:
Updated!